tests/Tests/Fixture/FixtureManager.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace OpenEMR\Tests\Fixture;
+
+require_once("library/sql.inc");
```

Comment:
to avoid issues, would prob use `dirname(__FILE__)` in the path

Reply:
note this requirement will likely go away when we get the bootstrapping of globals.php dealt with